Sources/Prorsum/HTTP/Serializer/RequestSerializer.swift

```diff
@@ -19,7 +19,23 @@ public struct RequestSerializer {
         let newLine: [UInt8] = [13, 10]
         
         let method = "\(request.method)".uppercased()
-        try stream.write("\(method) \(request.url.absoluteString) HTTP/\(request.version.major).\(request.version.minor)".bytes, deadline: deadline)
+
+        // if host is set
+        // build uri from path & query
+        // else use absoluteString
+        //
+        let request_uri: String
```

Comment:
Variable name should be camel case.
See https://swift.org/documentation/api-design-guidelines/